lib/components/src/blocks/IFrame.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import window from 'global';
+
+interface IFrameProps {
+  id: string;
+  key?: string;
+  title: string;
+  src: string;
+  allowFullScreen: boolean;
+  scale: number;
+  style?: any;
+}
+
+interface BodyStyle {
+  width: string;
+  height: string;
+  transform: string;
+  transformOrigin: string;
+}
+
+export class IFrame extends React.Component<IFrameProps> {
+  iframe: any = null;
+
+  componentDidMount() {
+    const { id } = this.props;
+    this.iframe = window.document.getElementById(id);
+  }
+
+  shouldComponentUpdate(nextProps: IFrameProps) {
+    const { src, scale } = nextProps;
```

Comment:
Do we need to handle the id changing here?

Reply:
Not AFAIK. Went through this in some detail with @ndelangen